action.yml

```diff
@@ -12,6 +12,14 @@ inputs:
   token:
     description: 'Ketch token'
     required: true
+  with_url:
+    description: 'With url prefix to Ketch Rest'
+    required: false
+    default: ''
```

Comment:
It would be better to change this to `url` and specify the default as the default rest URL.